docs/integration/announcements.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Announcements
+description: Details of the announcements functionality  
+---
+
+# Announcements
+
+Announcements are either Text-to-Speech or audio files that are sent to the MA players via the [`tts.xxxxxx`](https://www.home-assistant.io/integrations/tts/) or [`mass.play_announcement`](../faq/massannounce.md) HA actions. 
+
+## Overview
+
+If music is playing this will be temporarily paused and restored after the announcement.
+
+Players will be powered on before the announcement and then off again if it was not powered/active before the announcement.
+
+Volume can be adjusted while playing the announcement.
+
+If a player type has native support for announcements (e.g. Sonos) then that will be used in preference to the MA functionality providing an even smoother experience.
+
+The functionality works with all players that are supported by Music Assistant.
+
+Each player has settings which allow for adjustment of certain aspects of the announcement playback. Groups only have a setting for the pre-announcement sound.
+
+![image](../assets/screenshots/announcement-settings.png)
+
+## Native Support
+
+If an announcement is sent to a single player that supports native announcements, the announcement will be handled by the player itself.
+
+Native announcement support usually means that the player can "overlay" the announcement on top of any music that is playing. It will "duck" the volume of the music, play the announcement with a volume boost and then revert the music volume. Music keeps playing and the entire announcement will be handled by the player itself.
+
+Currently the only (known) player type that supports native announcements is Sonos S2 speakers.  Hopefully more player types will follow.
+
+## MA Player Support
+
+If a player does not support native announcements, MA will take care of it by stopping any playing music, adjust the volume (as configured), play the announcement, and then restore the previous state.
+
+!!! note
+    The MA announcement feature will ONLY work reliably if the player reports the state (e.g. playing, paused, idle) and the progress report (elapsed time) correctly. If any of those are not updated correctly then announcements will work poorly.
+
+ESPHome and Snapcast are known to be unreliable with state updates so expect issues with those player types. The same applies to some HA media players.
```

Comment:
Can we make this sound less harsh ?
For Snapcast I know @SantiagoSotoC is working on it to imrpove it. Also on the ESPHome side there is a new media player component in the making

Reply:
```suggestion
ESPHome and Snapcast can be unreliable with state updates so issues may occur with those player types. The same applies to some HA media players. Future updates of those platforms should improve performance.
```